_posts/2017-03-07-using-our-lead-submissions-api.markdown

```diff
@@ -60,59 +60,108 @@ request body is below:
-          "street2": "string",
-          "city": "string",
-          "state": "AK",
+          // The first line of the contact's current address
+          "street1": "36 Bromfield St",
+          // The second line of the contact's current address
+          "street2": "Second Floor",
+          // The city of the contact's current address
+          "city": "Boston",
+          // The state of the contact's current address
+          "state": "MA",
+          // The zipcode of the contact's current address
           "zip": "00112"
         }
+        // If there were additional contacts, they would be listed here
       }
     ],
+    // The account attributes object describes the business applying for a loan. 
+    // Since it is an object ({}), not a list ([]) there can only be one.
     "account_attributes": {
-      "entity_type": "Sole Proprietor",
-      "name": "string",
+      // The entity_type of the business, all valid options can be found on this page:
+      // https://api.forwardfinancing.com/api_docs
+      // Expand the lead endpoint and click on "Model"
+      "entity_type": "Limited Liability Company (LLC)",
+      // The name of the business, also known as the "DBA" or "Trade Name"
+      "name": "Pied Piper",
+      // The date the business was started
       "started_on": "2015-01-01",
-      "legal_name": "string",
+      // The legal name of the business
+      "legal_name": "Pied Piper LLC",
+      // The phone number of the business
       "phone": "6176781000",
-      "email": "test@forwardfinancing.com",
-      "website": "string",
-      "fein": "string",
+      // The email of the business
+      "email": "support@piedpiper.com",
+      // The website for the business
+      "website": "https://www.piedpiper.com/",
+      // The Federal Employer Identification Number for the business, with no spaces or dashes
+      "fein": "000000000",
+      // The average monthly revenue bucket for the business, valid options can be found on the documentation page
       "monthly_revenue": "Less than $5,000",
-      "industry_name": "Laundry and dry cleaning services",
+      // The name of the industry the business is in
+      "industry_name": "Compression Algorithms",
+      // The attributes for the businesses current address are nested within the "current_address_attributes"
       "current_address_attributes": {
-        "street1": "string",
-        "street2": "string",
-        "city": "string",
-        "state": "AK",
+        // The first line of the business's current address
+        "street1": "36 Bromfield St",
+        // The second line of the business's current address
+        "street2": "Second Floor",
+        // The city of the business's current address
+        "city": "Boston",
+        // The state of the business's current address
+        "state": "MA",
+        // The zipcode of the business's current address
         "zip": "00112"
       }
     },
     "loan_attributes": {
```

Comment:
@phereford What loan are we talking about for this object? The loan the merchant is applying for, or previous financing? Can you add a comment above the "loan_attributes" node explaining?